lib/storage.js

```diff
@@ -209,6 +209,60 @@ exports.set = function(key, json, callback) {
 };
 
 /**
+ * @summary Write multiple sets of user data
+ * @function
+ * @public
+ *
+ * @param {Object} keys
+ * @param {Function} callback - callback (error, done)
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.setMany({ foo: 'bar', bar: 'foo' }, function(error, done) {
+ *  if (error) {
+ *    console.log('Error saving ' + error.key);
+ *    throw error.error;
+ *  }
+ *  if (finished) console.log('finished');
+ * });
+ */
+exports.setMany = function(keys, callback) {
+  if (!keys || keys.constructor !== Object) {
+    callback(new Error('Invalid keys'), true);
+    return;
+  }
+
+  const entries = _.map(keys, function(value, key) {
+    return { key: key, value: value };
+  });
+
+  function process() {
+    const entry = entries.shift();
+    const key = entry.key;
+    const value = entry.value;
+
+    exports.set(key, value, function(error) {
+      if (error) {
+        callback({ error: error, key: key }, entries.length === 0);
+      }
+
+      if (entries.length > 0) {
+        process();
+      } else if (!error) {
+        callback(null, true);
+      }
+    });
+  }
+
+  if (entries.length > 0) {
+    process();
```

Comment:
I replied on #73 about using `async`, but the if statement is not necessary at all, it is just to ensure that in the event someone does `storage.setMany({})` that an error will not be thrown. I'm never used `async.parallel`  so I'm unsure if would be able to handle an event in which no keys are passed to the function.